frame-metadata/Cargo.toml

```diff
@@ -28,10 +28,17 @@ v12 = []
 v13 = []
 legacy = ["v13", "v12", "v11", "v10", "v9", "v8"]
 v14 = ["scale-info"]
-std = [
-	"codec/std",
-	"scale-info/std",
+
+full_derive = [
```

Comment:
Please add a comment for this feature to describe what it is for and what it enables. Also possibly could do with a slightly more accurate name? To me having `derive` there implies that it is supplying some derive macros.

Reply:
Good idea. Have broken them up into two features as some won't need serde.